common/app/experiments/Experiments.scala

```diff
@@ -17,10 +18,20 @@ object ActiveExperiments extends ExperimentsDefinition {
       HeaderTopBarSearchCapi,
       AdaptiveSite,
       OfferHttp3,
+      DeeplyRead,
     )
   implicit val canCheckExperiment = new CanCheckExperiment(this)
 }
 
+object DeeplyRead
+    extends Experiment(
+      name = "deeply-read",
+      description = "When ON, deeply read footer section is displayed",
+      owners = Seq(Owner.withGithub("@guardian/editorial-experience")),
```

Comment:
I wonder if this should be WebX now?

```suggestion
      owners = Seq(Owner.withGithub("@guardian/dotcom-platform")),
```